frontend/projects/ui/src/app/components/backup-drives/backup-drives.component.html

```diff
@@ -50,28 +76,48 @@ <h2 *ngIf="!cifs.mountable" class="inline">
       <!-- ** drives ** -->
       <ion-item-divider>Physical Drives</ion-item-divider>
       <!-- no drives -->
-      <ion-item *ngIf="!backupService.drives.length; else hasDrives" class="ion-padding-bottom">
+      <ion-item
+        *ngIf="!backupService.drives.length; else hasDrives"
+        class="ion-padding-bottom"
+      >
         <ion-label>
           <h2>
             <ion-text color="warning">
-              Warning! Plugging a 2nd physical drive directly into your Embassy can lead to data corruption.
+              Warning! Plugging a 2nd physical drive directly into your Embassy
+              can lead to data corruption.
             </ion-text>
           </h2>
           <br />
           <h2>
-            To backup to a physical drive, please follow the <a href="https://docs.start9.com/user-manual/general/backups.html#physical-drive" target="_blank" noreferrer>instructions</a>.
+            To backup to a physical drive, please follow the
+            <a
+              href="https://docs.start9.com/user-manual/general/backups.html#physical-drive"
```

Comment:
Same here
https://start9.com/latest/user-manual/backups/backup-create/#physical-backups

Reply:
Every link in the UI is a placeholder.

Reply:
Understood, but since these lines were edited anyway, and we know the scheme now, they could be correct.  I am prepared to do the full link update today or tomorrow latest.